include/highfive/bits/H5Node_traits.hpp

```diff
@@ -148,6 +148,23 @@ class NodeTraits {
     /// \param node_name The entry to check, path relative to the current group
     inline ObjectType getObjectType(const std::string& node_name) const;
 
+    template<typename Node,
+            typename std::enable_if<
+                std::is_same<Node, File>::value |
+                std::is_same<Node, Group>::value>::type* = nullptr>
+    Group createLink(
```

Comment:
I find `static_assert` better where it can be used. Readability and debugging ease

Reply:
Please pass `.clang-format`. style here is not consistent with rest

Reply:
Agree with clang format.
`enable_if` allows you to check whether your type is apropriate when you write code. If you try to pass inapropriate type then you will see that this line of code is underlined by red as I mistake. `static_assert` only throws error at a compile time so it is always a confusion what classes are accepted.